x-pack/plugins/lens/public/xy_visualization/xy_config_panel/threshold_panel.tsx

```diff
@@ -109,13 +109,94 @@ const IconSelect = ({
   );
 };
 
+function getIconPositionOptions({
+  isHorizontal,
+  axisMode,
+}: {
+  isHorizontal: boolean;
+  axisMode: YConfig['axisMode'];
+}) {
+  const options = [
+    {
+      id: `${idPrefix}auto`,
+      label: i18n.translate('xpack.lens.xyChart.thresholdMarker.auto', {
+        defaultMessage: 'Auto',
+      }),
+      'data-test-subj': 'lnsXY_markerPosition_auto',
+    },
+  ];
+  if (axisMode === 'bottom') {
+    const bottomOptions = [
+      {
+        id: `${idPrefix}above`,
+        label: !isHorizontal
+          ? i18n.translate('xpack.lens.xyChart.markerPosition.above', {
+              defaultMessage: 'Above',
+            })
+          : i18n.translate('xpack.lens.xyChart.markerPosition.right', {
+              defaultMessage: 'Right',
+            }),
+        'data-test-subj': 'lnsXY_markerPosition_above',
+      },
+      {
+        id: `${idPrefix}below`,
+        label: !isHorizontal
+          ? i18n.translate('xpack.lens.xyChart.markerPosition.below', {
+              defaultMessage: 'Below',
```

Comment:
Similar to previous suggestion, consider changing `Below` to `Bottom`.

```suggestion
              defaultMessage: 'Bottom',
```